docs/quarterly_RU_update.md

```diff
@@ -0,0 +1,44 @@
+#### A note on patch metadata
```

Comment:
This looks like a general look at patch metadata (which is fine);  can we name this file along those lines, and consistently with other files in this directory?  Thanks!

Reply:
Will change the file name from `docs/quarterly_RU_update.md` to `docs/patching-metadata.md` to match with other files names in:
```
12:33 $ ll docs/*
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan 134314 May  5 17:22 docs/bms-toolkit-architecture.png
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan   3194 May  5 17:22 docs/code-of-conduct.md
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan    327 May  5 17:22 docs/_config.yml
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan   1122 May  5 17:22 docs/contributing.md
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan    184 May  5 17:22 docs/index.html
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan   2178 Jun 30 14:23 docs/quarterly_RU_update.md
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan  75103 Jun 30 14:23 docs/user-guide.md

```